persistence/src/test/java/edu/unc/lib/dl/persist/services/edit/ExpireEmbargoServiceIT.java

```diff
@@ -0,0 +1,275 @@
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.ContextHierarchy;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+
+import static edu.unc.lib.dl.util.DateTimeUtil.formatDateToUTC;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextHierarchy({
+        @ContextConfiguration("/spring-test/test-fedora-container.xml"),
+        @ContextConfiguration("/spring-test/cdr-client-container.xml")
+})
+public class ExpireEmbargoServiceIT {
+
+    @Autowired
+    private String baseAddress;
+    @Autowired
+    private Model queryModel;
+    @Autowired
+    private FcrepoClient fcrepoClient;
+    @Autowired
+    private RepositoryObjectLoader repoObjLoader;
+    @Autowired
+    private RepositoryObjectFactory repoObjFactory;
+    @Mock
+    private OperationsMessageSender operationsMessageSender;
+    @Autowired
+    private TransactionManager txManager;
+    @Autowired
+    private SparqlQueryService sparqlQueryService;
+    @Autowired
+    private RepositoryObjectTreeIndexer treeIndexer;
+    @Captor
+    private ArgumentCaptor<List<PID>> pidListCaptor;
+
+    private ExpireEmbargoService service;
+
+    private ContentRootObject contentRoot;
+
+    private AdminUnit adminUnit;
+
+    private CollectionObject collObj1;
+    private CollectionObject collObj2;
+
+    @Before
+    public void init() throws Exception {
+        initMocks(this);
+        TestHelper.setContentBase(baseAddress);
+
+        service = new ExpireEmbargoService();
+        service.setOperationsMessageSender(operationsMessageSender);
+        service.setRepositoryObjectLoader(repoObjLoader);
+        service.setRepositoryObjectFactory(repoObjFactory);
+        service.setTransactionManager(txManager);
+        service.setSparqlQueryService(sparqlQueryService);
+
+        PID contentRootPid = RepositoryPaths.getContentRootPid();
+        try {
+            repoObjFactory.createContentRootObject(
+                    contentRootPid.getRepositoryUri(), null);
+        } catch (FedoraException e) {
+            // Ignore failure as the content root will already exist after first test
+        }
+        contentRoot = repoObjLoader.getContentRootObject(contentRootPid);
+    }
+
+    @Test
+    public void expireSingleEmbargoTest() throws Exception {
+        Calendar embargoUntil = yesterday();
+
+        createCollectionInUnit(new AclModelBuilder("Collection with embargo")
+                .addEmbargoUntil(embargoUntil)
+                .model);
+        PID pid = collObj1.getPid();
+        TestHelper.setContentBase(baseAddress);
```

Comment:
This is already being set in the `@Before` method, so you can remove these from the test methods.